src/control/cmd/dmg/pretty/storage_test.go

```diff
@@ -97,12 +106,7 @@ PCI:%s Model:%s FW:%s Socket:%d Capacity:%s
     Power Cycles:%d
     Power On Duration:%s
     Unsafe Shutdowns:%d
-    Error Count:%d
```

Comment:
this error count is specific to NVMe device stats right? ie not apart of BIO in-memory error counters I don't think. we can probably keep this in the display if so
Actually I am confused what this Error Count actually is?

Reply:
it's not used/populated anywhere . that's why I removed it, seems to be a surplus parameter which at some point was probably confused with  error log count